.travis.yml

```diff
@@ -1,15 +1,15 @@
 language: nix
 
-env:
-  NIX_PATH=nixpkgs=https://github.com/NixOS/nixpkgs/archive/c0b1e8a5fb174cd405dcca9f7fec275714ad9f4b.tar.gz
-
 install:
+  - source scripts/set_nixpath.sh
   - nix-env -iA nixopsUnstable -f '<nixpkgs>'
```

Comment:
also keep in mind, that the shellHook runs after nix-shell has eval'd a nix expression, so the nix file itself has to obey `nixpkgs-src.json` and not `NIX_PATH`